1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.js

```diff
@@ -11,11 +11,10 @@ Let's rewrite it (or _refactor_ it, as professional developers would call it):
 function doubleEvenNumbers(numbers) {
   // TODO rewrite the function body using `map` and `filter`.
   const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
+  const result = numbers
+    .filter((number) => number % 2 === 0)
+    .map((num) => num * 2);
+  newNumbers.push(...result);
```

Comment:
Why do you need the `newNumber` array? Can't you just return the result?

Reply:
Right, I don't need it.